dashboard/app/dsl/level_group_dsl.rb

```diff
@@ -21,6 +21,14 @@ def level(name)
     @hash[:levels] << name
   end
 
+  def submittable(text)
```

Comment:
can these be defined with the shortcuts? (see the integer and string lines above and the implementation in BaseDSL. There's a boolean shortcut too)